### PR TITLE
Lj 24 header component

### PR DESCRIPTION
**Pull request by MarcoGoC**
# Description

Mock header while the alpha site finishes their work
[LJ24] Header component with search bar and placeholders for menu and breadcrums 

## Test Instructions

1. get the branch and run 
1. how to test...

## Meets Definition of Done

- [ ] Meets design spec
- [ ] unit tests are included

### CODE REVIEW

File: components/atoms/Meta.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <title>{title}</title>
+    </Head>
+  );
+}
+
+Meta.defaultProps = {
+  title: "F1 Stats - Test App",
+  keywords: "formula 1, statistics, testing app",
+  description: "Stats from the Formula 1 circus",
+};
```
Comment (pcraig3) [inline components/atoms/Meta.js:25]: We should probably change this here ;)

**Review by pcraig3 — CHANGES_REQUESTED** [review]
Overall it looks pretty good, but I have 2 main comments after running it.

## on desktop 

| canada.ca | life journeys |
|---------|--------|
|  Screen Shot 2021-04-15 at 09 41 39   |   Screen Shot 2021-04-15 at 09 41 31  |
|   "francais" link is aligned with the search bar      |  "francais" link is not aligned with the search bar      |


| canada.ca | life journeys |
|---------|--------|
|   Screen Shot 2021-04-15 at 09 41 54    |  Screen Shot 2021-04-15 at 09 42 04   |
|    gov logo pretty large, on same row as french link, and the search bar is full-width     |  gov logo is really small, not on the same row as french link, search bar is not full width      |

I don't think it matters if we make the header link say "FR" on a small screen, but we can polish the layout a bit.

File: components/atoms/Meta.js (hunk 1 above)
Comment (MarcoGoC) [reply components/atoms/Meta.js:25]: what?,  I copy that from wrong repo

**Review by pcraig3 — APPROVED** [review]
Okay, I'm good with merging this as-is, and then updating the search bar later.

**Pull request merged**